### ISSUE TITLE
Can't load compressed cue+bin.

### ISSUE THREAD

**Issue by ofry** [libretro/RetroArch#4094]
Test case:

Any PSX game with cue+bin or cue+many bins in same archive. (example - from redump set).

If I try to open archive as folder and load *.cue from it, then:
1) Extracted to cache folder ONLY this cue, not bin.
2) When core tried to load *.bin, it will crash, because it isn't present in cache dir.

**Comment by bootsector** [comment]
Have you checked if the referenced .bin files have the correct case inside the .cue file?

**Comment by ofry** [comment]
Yes. And my OS is Windows 8.1 x64, so FILE.bin and File.bin and file.bin is same filename.

**Comment by bparker06** [comment]
You can currently only load .bin files from an archive, which means that multi-disc/track games are not supported this way. If you choose the cue file instead of the bin, it's not smart enough to extract the other file(s) yet.

**Comment by retro-wertz** [comment]
this is a similar behavior with the issue i posted in pce-cd, it will only extract the cue sheet but not the iso/bin

**Comment by lasers** [comment]
Linking retro-wertz's issue: https://github.com/libretro/RetroArch/issues/4003

**Comment by fr500** [comment]
It's not supported... compressing cue+bins is rather pointless anyway since extracting would take a looooong time and use quite a bit of ram and would need to be performed every single time you load the game.

Also what if a user has 100 roms (goodsets for instance) and loads one. We either extract everything from the zip or we write a cue parser.

And then what if someone want's to use the toc format or any other format.

And what about a multi cue + multi bin m3u.

Supporting every possible combination of things under the sun won't help us. It will make the codebase huge and riddled of bugs.

**Comment by ofry** [comment]
Redump sets have format:
game.zip => (cue + bins for this cue)
or
game (disc 1).zip => (cue + bins for disc 1 only).

So if user is pirate and uploading from his PC, he ought to have compressed and uncompressed sets either :( More than double disk space usage :(

**Comment by fr500** [comment]
My point stands.

**Comment by ofry** [comment]
And I think CUE and M3U parsers will help to create playlists from RA.

**Comment by ofry** [comment]
Maybe, if VFS will merge, then if core "asks" more files, then try to get it same way as initial file.
